source/grid/manifold_lib.cc

```diff
@@ -291,7 +291,9 @@ get_new_point (const ArrayView<const Point<spacedim>> &vertices,
     }
 
   // Unit norm direction.
-  candidate /= candidate.norm();
+  const double norm = candidate.norm();
+  if (norm > 0)
```

Comment:
but should not it throw an assert instead, namely `ExcDivideByZero`?

Reply:
I do not think so, because if norm is 0 that means candidate is `Point<dim> ()` and then the function will return `center`, which is correct.
I could alternatively write this as
```
if (norm == 0)
  return center;

candidate /= norm;
return ...
```
Would you prefer that?

Reply:
ah, i see. Yes, that probably is easy to read. Would you please also add a simple test to make sure this stays working and a change log for a bugfix?